Metalama.Framework.Engine/Advising/OverrideFinalizerAdvice.cs

```diff
@@ -14,6 +14,8 @@ namespace Metalama.Framework.Engine.Advising
     // TODO: Check why this class is unused.
     // ReSharper disable once UnusedType.Global
 
+    // Because we are using OverrideMethodAdvice, but that does not return a correct AdviceKind.
```

Comment:
I take it this is going to be fixed in the followup?

Reply:
Yep, I'm going to create a bug for that.